File: Data/Models/Pontuacao.cs
namespace TrabalhoAPI.Data.Models
{
    public class Pontuacao
    {
        public int Id { get; set; }
        public double Score { get; set; }
        public int UsuarioId { get; set; }
        public Usuario? Usuario { get; set; }
        public int FilmeId { get; set; }
        public Filme? Filme { get; set; }
    }
}

File: src/app/models/Pontuacao.ts
import { Filme } from "./Filme";
import { Usuario } from "./Usuario";

export class Pontuacao{
    id: number = 0;
    score: number = 0.0;
    usuarioId: number = 0;
    usuario: Usuario | undefined;
    filmeId: number = 0;
    filme: Filme | undefined;
}